deployments/wfirst-sit-dev/hubploy.yaml

```diff
@@ -0,0 +1,17 @@
+images:
```

Comment:
I'd like to understand what the purpose of the '-dev' deployment would be. Is it for individual people to hack on? Experimental tests to wfirst-sit that may or may not make it to prod? 

The problem with this is that things will drift apart between wfirst-sit and wfirst-sit-dev. So things that work in wfirst-sit-dev will not work in wfirst-sit, and vice versa. So depending on the reason for having a separate -dev deployment, let's see what is a good solution for it? It might be another 'dev' branch, a way for people to test this locally, deploy from your own fork, etc. But I'd like to avoid having '-dev' variants of each hub, for the drift reasons.

Reply:
This drift is already happening - for example, https://github.com/spacetelescope/jupyterhub-deploy/pull/3 added a `WORKDIR` to the Dockerfile, which is missing here.